CHANGES.md

```diff
@@ -51,15 +51,16 @@ To be released.
 
 ### Added APIs
 
- -  Added `KeyBytes` readonly struct.  [[#1678]]
- -  Added `HashDigest<T>(in ImmutableArray<byte>)` constructor.  [[#1678]]
- -  Added `HashDigest<T>.DeriveFrom(byte[])` overloaded static method. [[#1680]]
- -  Added `HashDigest<T>.DeriveFrom(ImmutableArray<byte>)` overloaded static
+ - Added `KeyBytes` readonly struct.  [[#1678]]
+ - Added `HashDigest<T>(in ImmutableArray<byte>)` constructor.  [[#1678]]
+ - Added `HashDigest<T>.DeriveFrom(byte[])` overloaded static method. [[#1680]]
+ - Added `HashDigest<T>.DeriveFrom(ImmutableArray<byte>)` overloaded static
     method. [[#1680]]
- -  Added `HashDigest<T>.DeriveFrom(ReadOnlySpan<byte>)` overloaded static
+ - Added `HashDigest<T>.DeriveFrom(ReadOnlySpan<byte>)` overloaded static
     method. [[#1680]]
- -  Added `StateStoreExtensions.EncodeKey()` static method.  [[#1689]]
- -  Added `StateStoreExtensions.DecodeKey()` static method.  [[#1689]]
+ - Added `StateStoreExtensions.EncodeKey()` static method.  [[#1689]]
+ - Added `StateStoreExtensions.DecodeKey()` static method.  [[#1689]]
+ - Added `TrieStateStsore.CopyStates` method.  [[#1691]]
```

Comment:
Space changes seem unintended.

Reply:
Oops, you're right. I'll revert these changes. 🙇🏻

Reply:
I amended!